pages/event.php

```diff
@@ -227,6 +227,9 @@ function event() {
                     break;
                 case "red_rep":
                     $player->getInventory();
+                    if($player->calMaxRepGain(1) == 0) {
```

Comment:
```suggestion
                    if($player->weekly_rep >= $player->weekly_rep_cap) {
```
Hmm couldn't we also do this?